docling/backend/mspowerpoint_backend.py

```diff
@@ -392,9 +392,10 @@ def handle_shapes(shape, parent_slide, slide_ind, doc, slide_size):
                     self.handle_tables(shape, parent_slide, slide_ind, doc, slide_size)
                 if shape.shape_type == MSO_SHAPE_TYPE.PICTURE:
                     # Handle Pictures
-                    self.handle_pictures(
-                        shape, parent_slide, slide_ind, doc, slide_size
-                    )
+                    if hasattr(shape, "image"): # make sure the Picture shape has an image attribute
+                        image_part = shape.image # get the image part
+                        if image_part.ext not in ["emf", "wmf"]: # all extensions except emf and wmf 
```

Comment:
Instead of excluding EMF and WMF pictures, this PR would better adopt how we handled it on the MS Word backend: https://github.com/docling-project/docling/blob/main/docling/backend/msword_backend.py#L671-L686

On Windows platforms, EMF and WMF actually work with PIL.

Reply:
@benichou Let us know when you expect this to be done! Love the work!

Reply:
Thanks @PeterStaar-IBM for the gentle reminder! I have been busier than usual at work but I have prepared the code this weekend and working on it today, you should have it today!